src/logger.js

```diff
@@ -0,0 +1,7 @@
+import bunyan from 'bunyan';
+
+export default bunyan.createLogger({
+  name: 'AddonValidatorJS',
+  stream: process.stdout,
+  level: process.env.LOG_LEVEL || 'fatal',
```

Comment:
I'll do this as a follow-up just to save context-switching. I've added https://github.com/mozilla/addons-validator/issues/100 so it doesn't get forgotten.